java/memory/memory-core/src/main/java/org/apache/arrow/memory/util/MemoryUtil.java

```diff
@@ -104,10 +103,7 @@ public Object run() {
                   constructor.setAccessible(true);
                   logger.debug("Constructor for direct buffer found and made accessible");
                   return constructor;
-                } catch (NoSuchMethodException e) {
-                  logger.debug("Cannot get constructor for direct buffer allocation", e);
-                  return e;
-                } catch (SecurityException e) {
+                } catch (Exception e) {
```

Comment:
I think the changes in this file can be reverted back to the original, it was made because of netty, see https://github.com/sirensolutions/arrow/pull/17

Reply:
@GeorgeAp I think you are referring to this [comment](https://github.com/sirensolutions/arrow/pull/50#discussion_r1570305497)?

Reply:
Yes it is the same comment, do you still get the error since there has been some changes after the above ?

Reply:
@GeorgeAp yes it still leads to the same error.

`Caused by: java.lang.reflect.InaccessibleObjectException: Unable to make private java.nio.DirectByteBuffer(long,long) accessible: module java.base does not "opens java.nio" to unnamed module @4b9af9a9`